src/visitors/Tag.js

```diff
@@ -35,22 +38,38 @@ function getAttributes(node: Object, context: Context): Array<JSXAttribute | JSX
           name = 'className';
           break;
       }
-      const expr = parseExpression(val === true ? 'true' : val, context);
-      if (!mustEscape && (!t.isStringLiteral(expr) || /(\<\>\&)/.test(val))) {
-        throw new Error('Unescaped attributes are not supported in react-pug');
+
+      let expr;
+
+      if (getInterpolationRefs(val)) {
```

Comment:
What if I did:

```js
const x = pug`div(foo="something${variable}somethingelse"
```

We need to verify that the interpolation represents the entire attribute value, and throw an error if not.